File: FSharp/FSharpRepoDB/FSharpRepoDB/Models/Person.fs
module FSharpRepoDB

type Person ( id: int64,
    name: string,
    age: int,
    address: string,
    isActive: bool ) =
    member this.Id = id
    member this.Name = name
    member this.Age = age
    member this.Address = address
    member this.IsActive = isActive

File: CSharp/AtomicBatchBulkOperations/AtomicBatchBulkOperations/Models/Person.cs
using AtomicBatchBulkOperations.Enumerations;
using System;

namespace AtomicBatchBulkOperations.Models
{
    public class Person
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public Gender? Gender { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public int Age { get; set; }
        public string ExtendedInfo { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedDateUtc { get; set; }
    }
}
